config/runtime.exs

```diff
@@ -6,6 +6,10 @@ import Config
 # and secrets from environment variables or elsewhere. Do not define
 # any compile-time configuration in here, as it won't be applied.
 # The block below contains prod specific runtime configuration.
+if System.get_env("PHX_SERVER") do
+  config :liveview_chat, LiveviewChat.Endpoint, server: true
```

Comment:
```suggestion
  config :liveview_chat, LiveviewChatWeb.Endpoint, server: true
```

Is the problem. :)

Reply:
Thanks again for your help and your time @jeregrine !